app/db/interface.py

```diff
@@ -135,6 +166,37 @@ def _query_payments(self, payment_attributes):
         except exc.SQLAlchemyError:
             raise PaymentServiceQueryError('Could not retrieve payment items due to query error with given attributes')
 
+# UTILITY FUNCTIONS
+    def is_valid_put(self,old_data,new_data):
+        valid = False
```

Comment:
Sounds good - is_default,is_removed,charge_history will not be part of payment_replacement. user_id will be a part of the input (similar to POST)
Also, there is a separate ticket #135 - for test cases for is_valid_put method. I will check if the request data has any NON UPDATABLE FIELDS, if so, it will raise a data validation error.